### DISCUSSION TITLE
Voice activation command on hero 11

### DISCUSSION THREAD

**Discussion by Anton778** [gopro/labs#1170] [Feature Requests]
On hero 9, there was a command to turn on the camera by voice, in subsequent models it was removed. 
It was very convenient to use it, because often your hands are busy driving the vehicle and you can't distract yourself to turn on the camera with the button. 

It would be great if you, the developers, would bring this team back to go pro 11 in a new firmware!

**Comment by dnewman-gpsw** [comment]
I thinking this is hardware issue, so nothing Labs firmware can do. I agree it was a cool feature, likely the battery impact was the reason the feature was removed.
